browser/src/control/Control.Menubar.js

```diff
@@ -169,7 +169,9 @@ L.Control.Menubar = L.Control.extend({
 					{uno: '.uno:InsertIndexesEntry'},
 					{uno: '.uno:InsertAuthoritiesEntry'},
 					{uno: '.uno:InsertMultiIndex'}]},
-				{uno: '.uno:InsertContentControl'},
+				{name: _UNO('.uno:ContentControlMenu', 'text'), type: 'menu', menu: [
```

Comment:
This is called .uno:ContentControlsMenu in core.git, and you need to use that name, otherwise you won't get a label from core. Could you please fix this?